ami/flowchart/library/DisplayWidgets.py

```diff
@@ -627,9 +632,10 @@ def data_updated(self, data):
                 v = np.rot90(v, self.rotate)
             if self.log_scale_histogram:
                 v = np.log10(v)
+            self.view.setAspectLocked(lock=self.lock, ratio=self.ratio)
 
             if v.any():
-                self.imageItem.setImage(v, autoLevels=self.auto_levels)
+                self.imageItem.setImage(v.T, autoLevels=self.auto_levels)
```

Comment:
I think this transposing conflicts with the change I made that forces pyqtgraph to default to row-major (aka c style)?

Reply:
I think this on the contrary needed because of that default row-major style. I have tested with the Jungfrau detector on some XPP datasets, and that looks correct. I should perhaps also go back to the RIX Andor to see if that messes things up there.